include/mpiInfo/mpiInfo.cpp

```diff
@@ -27,7 +27,15 @@
 #include <boost/program_options/variables_map.hpp>
 
 
-#define MPI_CHECK(cmd) {int error = cmd; if(error!=MPI_SUCCESS){printf("<%s>:%i ",__FILE__,__LINE__); throw std::runtime_error(std::string("[MPI] Error"));}}
+#define MPI_CHECK(cmd)                                                                                                \
```

Comment:
are these white spaces required by the new style?

Reply:
I think so that auto formated

Reply:
Okay? There are **96(!)** white spaces between `#define MPI_CHECK(cmd)` and the trailing `\`.